docs/raising-events.md

```diff
@@ -34,20 +34,33 @@ robot.FellInLove += Raise.WithEmpty(); // sender will be robot, args will be Eve
 robot.FellInLove += Raise.With(sender: robot, e: someEventArgs);
 ```
 
-Events of type `EventHandler<TEventArgs>` can be raised in exactly the same way. 
+Events of type `EventHandler<TEventArgs>` can be raised in exactly the same way.
 
-Note that as of version 3.1.0, for the .NET Standard version of the FakeItEasy library, `TEventArgs` need not extend `EventArgs` to support the above syntax. The .NET 4.0 library still requires `TEventArgs` to extend `EventArgs`.
+Note that as of version 3.1.0, for the .NET Standard and .NET 4.5 versions of the FakeItEasy library, `TEventArgs` need not extend `EventArgs` to support the above syntax. The .NET 4.0 library still requires `TEventArgs` to extend `EventArgs`.

```

Comment:
I simplified this paragraph. Since we're no longer mentioning a specific version, there's no point in saying that there's no constraint in 4.5 and netstd. We just need to say that there is a constraint in 4.0.